src/dotcallgraph.cpp

```diff
@@ -164,11 +164,16 @@ QCString DotCallGraph::getBaseName() const
 
 void DotCallGraph::computeTheGraph()
 {
+  QCString rankdir = Config_getString(DOT_RANKDIR_CALL);
+  if (rankdir.size() >= 2 && m_inverse)
+  {
+    std::swap(rankdir[0], rankdir[1]);
+  }
```

Comment:
How would you get 2 or more values on the rankdir part?
What happens with the values 3, 4, ...

Reply:
rankdir is supposed to be only "TB", "BT", "LR", or "RL"; this is supposed to swap the characters in the string: "TB" becomes "BT", etc.

My version of `dot` appeared to reset to its default (TB) if given an invalid string, e.g. "foo", so I saw no reason to explicitly check and replace when swapping does the same thing.

Reply:
Which version of `dot` do you use?

Reply:
```
dot -V
dot - graphviz version 2.43.0 (0)
```

I had tested the behavior with an invalid rankdir using a valid rankdir in DOT_GRAPH_ATTR and setting an invalid DOT_RANKDIR_INCLUDE.

Reply:
If rankdir is empty, the graph is not reversed with m_inverse true.  We could set rankdir to "LR" if its size is not 2, mentioning this in the config.xml documentation.